### PR TITLE
Add FB Open Graph tags for better sharing

### LINKED ISSUE #104: make the facebook card less awful
currently:
screen shot 2016-11-12 at 10 20 58 am

### PR DESCRIPTION
**Pull request by cmslewis**
#### Changes proposed in this pull request:

- Add Facebook [Open Graph tags](https://developers.facebook.com/docs/sharing/webmasters) to make the sharing artifact more attractive. 

#### Reviewers should focus on:

- @llorca @eparish - The images and text here are just placeholders. Feel free to change completely.
- There doesn't appear to be an easy way to test this before merging & releasing (at which point we can use Facebook's [Sharing Debugger](https://developers.facebook.com/tools/debug/sharing/?q=blueprintjs.com)).

### CODE REVIEW

File: packages/landing/src/index.html [hunk 1]
```diff
@@ -9,8 +9,19 @@
     <title>Blueprint – A React UI toolkit for the web</title>
     <meta name="description" content="A React UI toolkit for the web">
     <meta name="viewport" content="width=device-width">
+
+    <!-- Facebook Sharing -->
+    <meta property="og:url" content="http://blueprintjs.com/" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Blueprint - A React UI toolkit for the web" />
+    <meta property="og:description" content="Blueprint is a collection of elegant and easy-to-use React UI components from Palantir, freeing you to focus on building your product—-not the atomic pieces that comprise it." />
```
Comment (llorca) [inline packages/landing/src/index.html]: What about "A React UI toolkit for the web" to match the `<meta name="description">`?

Comment (cmslewis) [inline packages/landing/src/index.html]: Makes sense. Babel also has a fairly identical title and description for the card:
image

**Comment by llorca** [comment]
Was gonna suggest to put the logo straight up without a background, but I like what you did here 👍

**Comment by cmslewis** [comment]
Will merge now and debug once we release.

**Pull request merged**